feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/cloudBackup/RealLocalAccountsCloudBackupFacade.kt

```diff
@@ -279,6 +288,29 @@ class RealLocalAccountsCloudBackupFacade(
         ).filterNotNull()
     }
 
+    private fun CloudBackup.getAllAdditionalSecrets(walletPublicInfo: CloudBackup.WalletPublicInfo): Map<String, String> {
+        val privateInfo = privateData.wallets.findById(walletPublicInfo.walletId) ?: return emptyMap()
+        val chainAccountsSecretsByAccountId = privateInfo.chainAccounts.associateBy { it.accountId.intoKey() }
```

Comment:
We can move it inside getAllLedgerAdditionalSecrets, since it is not using outside